elasticdl/requirements.txt

```diff
@@ -2,4 +2,5 @@ grpcio-tools
 kubernetes==10.1.0
 docker
 pyrecordio>=0.0.6
+Cython
```

Comment:
PyODPS will compile some C code if Cython is installed.
Please refer to http://pyodps.alibaba.net/pyodps-docs/zh-CN/latest/base-tables.html#tunnel